test/integrations/destinations/mp/processor/data.ts

```diff
@@ -0,0 +1,6591 @@
+const defaultMockFns = () => {
+  jest.spyOn(Date, 'now').mockImplementation(() => new Date(Date.UTC(2020, 0, 25)).valueOf());
+};
+
+export const data = [
+  {
+    name: 'mp',
+    description: 'Test 0',
+    feature: 'processor',
+    module: 'destination',
+    version: 'v0',
+    input: {
+      request: {
+        body: [
+          {
+            destination: {
```

Comment:
Even we should reuse input also and only change what is different

Reply:
It can be done by defining a single template input (`input1`) and then redefining multiple inputs (`input2`, `input3`), with required changes and using it in the test case. Should we prefer it?

Reply:
we can keep one global input.
```js
{
input1: { ...globalInput, abc: 2}
}
```

Reply:
This will become complex I guess, as we have multiple nesting in the inputs and anyway we will have to redefine all the inputs also causing a readability issue. WDYT?

Reply:
what about we do this for destination config to start with?